credentials-considerations.md

```diff
@@ -1,38 +1,119 @@
+
+Some people are unwilling or unable to rely on government-issued identification. Stateless people, refugees, immigrants, unsheltered people and young people all may have difficulty in accessing government-issued credentials. Dependence on RWI systems may place these users at risk of increasingly being excluded from larger fractions of online activity.
+
+There are many people who prefer not to use the dominant operating systems or prefer to ensure they have total control over the devices they own. Such usage patterns may conflict with the security guarantees that many credential issuers will require prior to provisioning credentials, so a requirement to use such credentials may exclude such users.
+
+There's no universal agreement on what constitutes "adult content". One person's educational material may be someone else's "adult content". Well-meaning regulations and technical mechanisms to protect children may also be used to restrict the free flow of information with a particular political agenda. Similarly, content restricted to people who can prove their citizenship or residency in one set of countries or another may further accelerate the fragmention of the web into a "splinternet", with particular implications for regimes who are interested in restricting access to information. 
+
+* who may be excluded from online services by this technology and requirements? 
+* who will be discouraged from browsing or communicating by an age or credential requirement? (chilling effects)
+* how effective for the specific use cases are age verification or identity attribution presentation technologies? if they aren't fully effective, what is the trade-off between their harms and efficacy?
+* what bias may be present? 
+* how is remediation handled, when an estimate or presented characteristic is incorrect?
+* if age-gating or identity credentials are in use, to what level will they be required, and what are the impacts for people who do not have the technology or credentials, or are unwilling to use them? (discrimination)
+* will normal people be required to enroll in these systems?  to present them in certain contexts?
+* will a verifier be obliged to offer this form of identification/verification?  will the verifier be able to offer other forms of verification?
+* even if not formally mandatory, what social/economic/cultural pressures push in their direction?
+* what sort of pushback is available to people who do not participate?
+
+Privacy risks also have free expression implications here: who will be restricted from speaking or accessing information on the Web when there is a risk that a government-issued identity may be linked to that reading or writing?
 
-### Selective disclosure
+### Consolidation
```

Comment:
"Consolidation" may not be the right word for this section. "Identity System Dilemmas" is probably more accurate but it doesn't roll of the tongue.